src/deluge/dsp/compressor/rms_feedback.cpp

```diff
@@ -122,8 +140,8 @@ float RMSFeedbackCompressor::calcRMS(StereoSample* buffer, uint16_t numSamples)
 
 	} while (++thisSample != bufferEnd);
 
-	float ns = float(numSamples);
-	mean = (2 * float(sum) / ONE_Q31f) / ns;
+	float ns = float(numSamples * 2);
+	mean = (float(sum) / ONE_Q31f) / ns;
```

Comment:
@m-m-adams is this intentional? Now that I'm looking at the math again I'm pretty sure it's correct at least.